python/cuml/preprocessing/encoders.py

```diff
@@ -174,6 +174,20 @@ def _check_input(self, X, is_categories=False):
             self._set_input_type('df')
             return X
 
+    def _check_input_fit(self, X, is_categories=False):
+        """Helper function to be overridden by dask OneHotEncoder"""
```

Comment:
It's a little awkward to have references to subclass-specifics in a parent class. Ideally, this function would just say something like "Can be overridden in subclasses".

Reply:
Good point!